windows_containers/creating_windows_machinesets/creating-windows-machineset-nutanix.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+[id="creating-windows-machineset-nutanix"]
+= Creating a Windows MachineSet object on Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: creating-windows-machineset-nutanix
+
+toc::[]
+
+You can create a Windows `MachineSet` object to serve a specific purpose in your {product-title} cluster on Nutanix. For example, you might create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines.
+
+[discrete]
```

Comment:
I don't think this [discrete] tag is meant to be here. It's making the machine API overview not show in the TOC